index.bs

```diff
@@ -1239,10 +1243,13 @@ Gets the {{DOMException}} object passed to {{SensorErrorEventInit}}.
 
     : input
     :: |sensor_instance|, a {{Sensor}} object.
-    :: |options|, a {{SensorOptions}} object.
+    :: |options|, a [=dictionary=] object.
     : output
     :: None
 
+    1.  [=map/For each=] |key| → <var ignore>value</var> of |options|
+        1.  If the associated [=sensor supported options=] [=set/contains|does not contain=] |key|
+            1. [=Throw=] "{{NotSupportedError!!exception}}" {{DOMException}}.
```

Comment:
What if the value is invalid(e.g. out of range, invalid type), shell we throw exception as well? Current implementation in Chrome throws `TypeError`.

Reply:
As for values, that is to be specified separately for each supported property. 
`TypeError` is send for  the `referenceFrame` attribute, as its values are enum members, so it is controlled on IDL level. 
The `NotSupportedError` exception is to be used specifically for the feature detection.